library/src/hooks/places.ts

```diff
@@ -1,12 +1,12 @@
-import {useMemo} from 'react';
+import {useContext, useMemo} from 'react';
 
-import {useGoogleMap} from './map-instance';
+import {GoogleMapsContext} from '../google-maps-provider';
 
 /**
  * Hook to get Google Maps Places Service instance
  */
 export const usePlacesService = (): google.maps.places.PlacesService | null => {
-  const {map} = useGoogleMap();
+  const {map} = useContext(GoogleMapsContext);
```

Comment:
So does this mean we cannot use the `useGoogleMap` hook anymore? 

Or can we use this here:
`const {map} = useGoogleMap();`

Reply:
Sure we can use `const map = useGoogleMap();`. I just thought it would be more uniform to use the `useContext` hook everywhere in our hooks.

Reply:
Ok got it 👍